qiskit/transpiler/__init__.py

```diff
@@ -684,10 +682,12 @@
   :class:`~.SabreLayout` is used to select a layout if a perfect layout isn't found for
   optimization levels 1, 2, and 3.
 - :class:`~.TrivialLayout`: Always used for the layout at optimization level 0.
+
+There are other passes than can be used for the heuristic stage, but are not included in the default
+pipeline, such as:
```

Comment:
I know this already merged but how does this reconcile with: https://github.com/Qiskit/qiskit/pull/12453 which is new in 1.2.0? We're not running the "pass" by default. But we're using the pass internals as part of another one.